python/ray/air/_internal/torch_utils.py

```diff
@@ -8,7 +8,9 @@
 
 import ray
 from ray.air.util.data_batch_conversion import _unwrap_ndarray_object_type_if_needed
-
+from ray._private.accelerators.hpu import HPU_PACKAGE_AVAILABLE
```

Comment:
For changes in this file, let's do it in a separate file.

Reply:
can you elaborate ? You mean imports from another import ?

Reply:
Sorry, had a typo. I mean for changes in python/ray/air, lets' do it in a separate PR.